src/controllers/expenses.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const expenseServise = require('../services/expenses.js');
+const userServise = require('../services/users.js');
+
+let nextId = 0;
+
+function add(req, res) {
+  const {
+    userId,
+    title,
+  } = req.body;
+
+  if (!title || !userServise.exist(userId)) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = {
+    ...req.body,
+    id: nextId++,
```

Comment:
Check is id this way works properly? Maybe better use `Math.random with new Date` and count it here. No initializing in line 6(because as I understand, it doesn't work like with closure). (We could discuss it 🙂 )